### PR TITLE
Added a half birthday blog post, as discussed on dev@.

### PR DESCRIPTION
**Pull request by francesperry**
R: @jbonofre @evilsoapbox 

Let me know what you think! Incorporated comments from the dev@ thread, though I erred towards not listing too many things that are just in discussion.

I'm also not totally sure how future post dates work. I had to run 'jekyll serve --future' to see it staged locally. Don't quite follow what happens when it eventually gets submitted.

### CODE REVIEW

**Comment by jbonofre** [comment]
The link is OK in the PR. Only the `2016-08-04-six-months.md` is pending.

File: _posts/2016-08-04-six-months.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (francesperry) [inline _posts/2016-08-04-six-months.md]: Whoops. git snafu. fixed.

File: _posts/2016-08-04-six-months.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+And perhaps most importantly, we’re committed to building an involved, welcoming community. So far, we’ve:
+
+* Started building a vibrant developer community, with detailed design discussions on features like DoFn reuse semantics, serialization technology, and an API for accessing state.
+* Started building a user community with improvements to the website and documentation.
+* Had multiple talks on Beam at venues including ApacheCon, Hadoop Summit, Kafka Summit, JBCN Barcelona, and Strata.
+* Presented at multiple existing meetups and are starting to organize some of our own.
+
+While it’s nice to reflect back on all we’ve done, we’re working full steam ahead towards a stable release and graduation from incubator. And we’d love your help -- join the [mailing lists](http://beam.incubator.apache.org/use/mailing-lists/), check out the [contribution guide](http://beam.incubator.apache.org/contribute/contribution-guide/), and grab a [starter task](https://issues.apache.org/jira/browse/BEAM-342?jql=project%3DBEAM%20AND%20text%20~%20%22starter%22) from Jira!
```
Comment (swegner) [inline _posts/2016-08-04-six-months.md]: Filter out resolved starter tasks. I think this query works: https://issues.apache.org/jira/browse/BEAM-442?jql=project%20%3D%20BEAM%20AND%20status%20%3D%20Open%20AND%20text%20~%20%22starter%22

**Comment by evilsoapbox** [comment]
With respect to the time - Jekyll will not "build" content until the post date is after the real date. This is kind of a weird quirk because we have to build the static content beforehand and having Jekyll perform date math is a bit annoying. To fix for the interim:

1. Set date for the post to a date previous to today
2. Review post
3. Update to the desired time
4. **After** that time has passed, regenerate (build) the site and squash the commits
5. Merge
6. Profit, way too late.

File: _posts/2016-08-04-six-months.md (hunk 2 above)
Comment (francesperry) [inline _posts/2016-08-04-six-months.md]: using label, not text:
https://issues.apache.org/jira/browse/BEAM-520?jql=project%20%3D%20BEAM%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20in%20(newbie%2C%20starter)

File: _posts/2016-08-04-six-months.md [hunk 3]
```diff
@@ -18,26 +18,26 @@ Looking just at raw numbers for those first six months, that’s:
...
-But behind those numbers, there’s been a ton of technical progress, including:
+And behind those numbers, there’s been a ton of technical progress, including:
 
 * Refactoring of the entire codebase, examples, and tests to be truly runner-independent.
 * New functionality in the Apache Flink runner for timestamps/windows in batch and bounded sources and side inputs in streaming mode.
 * Work in progress to upgrade the Apache Spark runner to use Spark 2.0.
 * Several new runners from the wider Apache community -- Apache Gearpump has its own feature branch, Apache Apex has a PR, and conversations are starting on Apache Storm and others.
-* New SDKs/DSLs -- the Python SDK from Google is in, and there are plans to add the Scio DSL from Spotify.
-* Support for new IO connectors -- Apache Kafka and JMS are in, with Amazon Kinesis, Apache Cassandra, and MongoDB in PR, and more in planning.
+* New SDKs/DSLs for exposing the Beam model -- the Python SDK from Google is in on a feature branch, and there are plans to add the Scio DSL from Spotify.
```
Comment (evilsoapbox) [inline _posts/2016-08-04-six-months.md]: Should we call out the branch by name?

File: _posts/2016-08-04-six-months.md [hunk 4]
```diff
@@ -18,26 +18,26 @@ Looking just at raw numbers for those first six months, that’s:
...
+* Support for additional data sources and sinks -- Apache Kafka and JMS are in, there are PRs for Amazon Kinesis, Apache Cassandra, and MongoDB, and more connectors are being planned.
```
Comment (evilsoapbox) [inline _posts/2016-08-04-six-months.md]: Same as above, would links help or be a waste of time?

**Comment by dhalperi** [comment]
LGTM.

R: @dhalperi to merge, per our conversation.

File: _posts/2016-08-04-six-months.md [hunk 5]
```diff
@@ -18,26 +18,26 @@ Looking just at raw numbers for those first six months, that’s:
...
 * Presented at multiple existing meetups and are starting to organize some of our own.
 
-While it’s nice to reflect back on all we’ve done, we’re working full steam ahead towards a stable release and graduation from incubator. And we’d love your help -- join the [mailing lists](http://beam.incubator.apache.org/use/mailing-lists/), check out the [contribution guide](http://beam.incubator.apache.org/contribute/contribution-guide/), and grab a [starter task](https://issues.apache.org/jira/browse/BEAM-342?jql=project%3DBEAM%20AND%20text%20~%20%22starter%22) from Jira!
+While it’s nice to reflect back on all we’ve done, we’re working full _stream_ ahead towards a stable release and graduation from incubator. And we’d love your help -- join the [mailing lists](http://beam.incubator.apache.org/use/mailing-lists/), check out the [contribution guide](http://beam.incubator.apache.org/contribute/contribution-guide/), and grab a [starter task](https://issues.apache.org/jira/browse/BEAM-520?jql=project%20%3D%20BEAM%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20in%20(newbie%2C%20starter)) from Jira!
```
Comment (evilsoapbox) [inline _posts/2016-08-04-six-months.md]: Minor editing nit - would this better be expressed in an enumerated list? Just an idea, not a major deal. :)

File: _posts/2016-08-04-six-months.md (hunk 4 above)
Comment (francesperry) [inline _posts/2016-08-04-six-months.md]: Sorry these didn't get addressed. I decided to limit the links in the post itself -- every line would have linked to something. Felt like it was better for the final few links to pop.

**Comment by dhalperi** [comment]
Hmm, I somehow failed to add the merge commit that closed this PR. Can you close it manually please, @francesperry ?

**Comment by francesperry** [comment]
Merged.

**Pull request closed without merging**